docs/ref/checks.txt

```diff
@@ -159,6 +159,8 @@ Model fields
 * **fields.W161**: Fixed default value provided.
 * **fields.W162**: ``<database>`` does not support a database index on
   ``<field data type>`` columns.
+* **fields.E170**: ``BinaryField`` ``default`` cannot be a string, use bytes
```

Comment:
Is there any reason not to use `E163`? :thinking:

Reply:
It was on the initial patch. so I thought it has special meaning.

Reply:
@claudep Thanks :+1:, now it makes sense for me :smile: 

@hramezani Please switch to the `E170`.

Reply:
@claudep, @felixxm, Done